internal/completion/kubernetes.go

```diff
@@ -0,0 +1,28 @@
+package completion
+
+import (
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/service"
+
+	"github.com/UpCloudLtd/upcloud-go-api/v4/upcloud/request"
+	"github.com/spf13/cobra"
+)
+
+// Kubernetes implements argument completion for load balancers, by uuid or name.
```

Comment:
`for load balancers` → `for kubernetes clusters`

Reply:
Fixed in https://github.com/UpCloudLtd/upcloud-cli/pull/190/commits/5d5ce9c85dcfbba240bd47a1f1d887e5f2679bbd